predicates/auth/jwt.go

```diff
@@ -0,0 +1,183 @@
+/*
+Package auth implements custom predicates to match based on content
+of the HTTP Authorization header.
+
+You can use this predicate to match a route based on data in the 2nd part of a JWT token, for example based on the issuer.
```

Comment:
I think that it is nicer to use the passive form of "This predicate can be used..." in code docs.